### ISSUE TITLE
Check that instructions are styled properly

### ISSUE THREAD

**Issue by Angelsaur** [alan-turing-institute/AutSPACEs#44]
### Summary
You need to make sure you’re not conveying instructions in a way that relies on a user’s ability to see the page

### Deadline 

20/02/2021

### What needs to be done?

You need to make sure you’re not conveying instructions in a way that relies on a user’s ability to see the page.

For example, only sighted users will understand instructions like:

‘click the round button’
‘click the big button below’
‘click the red button’
Users who cannot see the page will not know what you’re referring to, because instructions like that rely on visual descriptions.

So check your sample of pages and look out for these types of instructions

**Comment by gedankenstuecke** [comment]
Made issue #369 to create an accessibility checklist that can be used for the MVP and going forward. Closing this along with other accessibility checklist items for now.
